calico_cni/policy_drivers.py

```diff
@@ -367,14 +385,25 @@ def get_policy_driver(k8s_pod_name, k8s_namespace, net_config):
         sys.exit(ERR_CODE_GENERIC)
 
     # Determine which policy driver to use.
-    if k8s_pod_name:
+    if cni_plugin.k8s_pod_name:
```

Comment:
Now that we have the whole cni_plugin object, let's use this:
https://github.com/projectcalico/calico-cni/blob/master/calico.py#L146